src/rpcserver.cpp

```diff
@@ -282,6 +282,9 @@ static const CRPCCommand vRPCCommands[] =
     { "network",            "getpeerinfo",            &getpeerinfo,            true  },
     { "network",            "ping",                   &ping,                   true  },
 
+    /* ZMQ notification */
+    { "zmq",                "getzmqurl",              &getzmqurl,              true  },
```

Comment:
What's the use case of `getzmqurl`? Why should someone need to get the ZMQ URL over RPC?